app/assets/javascripts/application.js.coffee

```diff
@@ -197,3 +197,9 @@ $ ->
     new ConfirmDangerModal(form, text)
 
   new Aside()
+
+  $('.hide-project-limit-message').on 'click', (e) ->
+    path = '/'
+    $.cookie('hide_project_limit_message', 'false', { path: path })
+    $(@).parents('.project-limit-message').remove()
+    e.preventDefault()
```

Comment:
The new code is rendered in partial `_project_limit_message.html.haml`.  I'm specifically talking about placement of the Javascript.  Currently, similar Javascript for the no ssh key message resides in [project.js.coffee](https://github.com/gitlabhq/gitlabhq/blob/master/app/assets/javascripts/project.js.coffee#L22).  However, this code doesn't seem to execute on the dashboard. Unfortunately, I'm not entirely sure how Javascript execution is determined in GitLab so I wasn't sure which file was best suited for this piece of code and shoved it in `application.js.coffee`.